input/pagecontent/ValueSet-mcode-observation-codes-distant-metastases-vs-intro.md

```diff
@@ -0,0 +1,3 @@
+### Usage
+
+The codes in this value set designate the staging observation as either clinical, pathological, or other. In AJCC terminology, clinical staging is represented by "c" or "yc" prefix, referring to staging done before first treatment, or staging done post-therapy or post-neoadjuvant therapy but before surgery. Pathological classification, represented by "p" or "yp" prefix, is used for patients if surgery is the first definitive therapy, or after neoadjuvant therapy and planned post-neoadjuvant therapy surgery. The "Other" code is meant to include staging at the time of recurrence or progression ("r" prefix), or for cancers not previously recognized and found as an incidental finding at autopsy ("a" prefix).
```

Comment:
I think we should use an include (like for the Markdown links) to avoid repeating the same content in multiple places.

Reply:
Put this on the "someday it would be nice" list. There is not that much repetition.